app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
Nghiên cứu I18n để sử dungj cho tất cả view nhìn thấy trên màn hình nhé

Reply:
`Ruby on Rails Tutorial Sample App` 
sử dụng `I18n in rails ` hì